index.html

```diff
@@ -543,6 +543,111 @@ <h6>JWT Decoding</h6>
+        There are currently 2 competing solutions to this problem described below. It is a goal
+        of the v2 work to resolve them and
+        reduce production rules to a single, simple, set of instructions that
+        any implementer can easily meet if they possess a software library
+        supporting [[RFC7515]] or [[RFC7519]].
+      </p>
+
+      <section>
+        <h3>In addition to...</h3>
+
+        <p>
+          There are several members (claims) of the
+          <code>Credential</code> which will need to be translated to their JOSE
+          form, and included next to the <code>vc</code> or
+          <code>vp</code> member in the <code>payload</code>.
+        </p>
+
+        <p>
+          If a member is not present in the <code>Credential</code> it MUST NOT
+          be present in the <code>VerifiableCredential</code> as either a claim
+          in the payload or a claim in the <code>vc</code> attribute of the
+          payload.
+        </p>
+
+        <p>
+          We start with an empty header, and payload objects, and we add members
+          to the header and the payload based on the content in the
+          <code>Credential</code>.
+        </p>
+
+        <section>
+          <h4>issuer -> iss</h4>
+
+          <p>
+            This member MUST be present in the
+            <code>payload.vc.issuer</code> attribute as either a string or an
+            <code>object</code> with and <code>id</code>.
+          </p>
+
+          <p>This member MUST be present <code>payload.iss</code>.</p>
+
+          <p>
+            In the case that <code>payload.vc.issuer</code> is an object,
+            <code>payload.iss</code> MUST be <code>payload.vc.issuer.id</code>.
+          </p>
+
+          <p>
+            In the case that
+            <code>payload.vc.issuer</code> is a string,
+            <code>payload.iss</code> must be
+            <code>payload.vc.issuer</code>
+          </p>
+        </section>
+
+        <section>
+          <h4>issuanceDate -> nbf</h4>
+
+          <p>
+            This member MUST be present in the
+            <code>payload.vc.issuanceDate</code> attribute as an XMLDateTime
+            String.
+          </p>
+
+          <p>
+            This member MUST be present <code>payload.nbf</code> as a unix
+            timestamp.
+          </p>
+
+          <p>
+            In the case that the <code>issuanceDate</code> includes leap
+            seconds, it is not possible to detect them when the date time is
+            represented in <code>nbf</code>
+          </p>
+        </section>
+
+        <p class="issue">This section needs to be defined.</p>
+
+        <section>
+          <h4>issuance</h4>
+
+          <p>
+            The header and payload converted into a JWT, in accorance with the
+            RFC:
+            <a href="https://www.rfc-editor.org/rfc/rfc7519#section-7.1"
+              >RFC7519 Section 7.1</a
+            >
+          </p>
+        </section>
```

Comment:
I don't understand what the issuance section is trying to convey.

Reply:
Happy to explain verbally.

This section closes the loop for readers regarding how a `header` and `payload` become a JWT.

We can't just magically assume that people know how to sign them properly.

Generally, the process is:

```
signature = base64url ( 
    rawSign ( 
       base64url (header) + '.' + base64url (payload) 
   )
)
```

See https://github.com/panva/jose/blob/8b9955515f6c81b322fe8603201c204de0bfbec2/src/jws/flattened/sign.ts#L129

Reply:
All right.  This is innocuous.  I just wasn't sure that it's necessary.

Reply:
I think its very necessary actually, especially related to `kid` guidance and hints on discovering the publicKey... but perhaps we can follow up on an issue, lets use: https://github.com/w3c/vc-jwt/issues/30